src/ripple/app/main/Main.cpp

```diff
@@ -242,7 +270,9 @@ runUnitTests(
 
         multi_runner_child child_runner{num_jobs, quiet, log};
         child_runner.arg(argument);
-        auto const any_failed = child_runner.run_multi(multi_selector(pattern));
+        multi_selector pred(pattern);
+        auto const any_failed =
+            child_runner.run_multi(pred) || anyMissing(child_runner, pred);
```

Comment:
I noticed that `run_multi` returns `true` on failure. Should call `anyMissing` to report any missing filters if `run_multi` fails?

Reply:
Oooh. I didn't think about the short-circuit. Good catch. I'll fix it so both functions are called every time.